disperser/controller/dispatcher.go

```diff
@@ -158,15 +165,24 @@ func (d *Dispatcher) HandleBatch(ctx context.Context) (chan core.SigningMessage,
 				DispersedAt:     uint64(time.Now().UnixNano()),
 				BatchHeader:     *batch.BatchHeader,
 			}
+			putDispersalRequestStart := time.Now()
 			err := d.blobMetadataStore.PutDispersalRequest(ctx, req)
 			if err != nil {
 				d.logger.Error("failed to put dispersal request", "err", err)
 				return
 			}
 
-			for i := 0; i < d.NumRequestRetries+1; i++ {
+			d.metrics.reportPutDispersalRequestLatency(time.Since(putDispersalRequestStart))
+
+			var i int
+			for i = 0; i < d.NumRequestRetries+1; i++ {
+				sendChunksStart := time.Now()
 				sig, err := d.sendChunks(ctx, client, batch)
 				if err == nil {
+
+					sendChunksFinished := time.Now()
+					d.metrics.reportSendChunksLatency(sendChunksFinished.Sub(sendChunksStart))
```

Comment:
I think we should report the latency for failed requests as well

Reply:
done (this location and elsewhere)